kubejs/server_scripts/base/entity/spawned/custom_occultism_spawns.js

```diff
@@ -0,0 +1,33 @@
+EntityEvents.spawned((event) => {
+    if (event.entity.type !== 'minecraft:item') {
+        return;
+    }
+
+    let item_type = event.entity.item.id.split(':')[1];
+
+    if (
+        event.entity.type == 'minecraft:item' &&
+        Object.keys(custom_spawns).includes(item_type) &&
+        event.entity.item != null &&
+        event.entity.item != undefined
```

Comment:
I belive you can just check for `event.entity.item`, it'll be false if it's null or undefined.
And preferably have that check as 2nd, since it's a pretty lightweight check compared to the Object.keys one.

Reply:
Unfortunately `event.entity.item` doesn't appear to resolve true when I drop an item. Not sure what's up there. 

I'll swap the logic around though, there's no reason to check the type twice and all those lighter weight checks can get put in the `return` check.

Reply:
err, woops, helps if I don't make typos when testing. Alright, cleaned up.